lib/generator._js

```diff
@@ -95,6 +96,18 @@ function ScriptGenerator(repositoryRoot, projectType, projectPath, solutionPath,
     var relativeSolutionPath = path.relative(repositoryRoot, solutionPath);
     log.info('Solution file path: .' + path.sep + relativeSolutionPath);
     this.solutionPath = relativeSolutionPath;
+
+    // find .xproj file, should be under projectPath
+    var files = fs.readdirSync(projectPath);
+    var webProjFiles = files.filter(function(file){
+      return file.endsWith(".xproj");
```

Comment:
@mejje, you can use `node -v` in kudu CMD to check for your node version, I suspect yours is prior to `v4.0.0`, otherwise it would work (you can see from the above test), #54  should run with all the  node versions under kudu `D:\Program Files (x86)\nodejs`